src/scripts/site.js

```diff
@@ -1,225 +1,337 @@
+            require(["serverRestartDialog", "events"], function (ServerRestartDialog, events) {
                 var dialog = new ServerRestartDialog({
                     apiClient: apiClient
                 });
-                events.on(dialog, "restarted", function() {
-                    AppInfo.isNativeApp ? apiClient.ensureWebSocket() : window.location.reload(!0)
-                }), dialog.show()
-            })
-        },
-        capabilities: function(appHost) {
-            var caps = {
-                PlayableMediaTypes: ["Audio", "Video"],
-                SupportedCommands: ["MoveUp", "MoveDown", "MoveLeft", "MoveRight", "PageUp", "PageDown", "PreviousLetter", "NextLetter", "ToggleOsd", "ToggleContextMenu", "Select", "Back", "SendKey", "SendString", "GoHome", "GoToSettings", "VolumeUp", "VolumeDown", "Mute", "Unmute", "ToggleMute", "SetVolume", "SetAudioStreamIndex", "SetSubtitleStreamIndex", "DisplayContent", "GoToSearch", "DisplayMessage", "SetRepeatMode", "ChannelUp", "ChannelDown", "PlayMediaSource", "PlayTrailers"],
-                SupportsPersistentIdentifier: "cordova" === self.appMode || "android" === self.appMode,
-                SupportsMediaControl: !0
-            };
-            caps.IconUrl = appHost.deviceIconUrl(), caps.SupportsSync = appHost.supports("sync"), caps.SupportsContentUploading = appHost.supports("cameraupload");
-            appHost.getPushTokenInfo();
-            return caps = Object.assign(caps, appHost.getPushTokenInfo())
+                events.on(dialog, "restarted", function () {
+                    if (AppInfo.isNativeApp) {
+                        apiClient.ensureWebSocket();
+                    } else {
+                        window.location.reload(true);
+                    }
+                });
+                dialog.show();
+            });
         }
     },
-    AppInfo = {};
-! function() {
+    capabilities: function (appHost) {
+        var caps = {
+            PlayableMediaTypes: ["Audio", "Video"],
+            SupportedCommands: ["MoveUp", "MoveDown", "MoveLeft", "MoveRight", "PageUp", "PageDown", "PreviousLetter", "NextLetter", "ToggleOsd", "ToggleContextMenu", "Select", "Back", "SendKey", "SendString", "GoHome", "GoToSettings", "VolumeUp", "VolumeDown", "Mute", "Unmute", "ToggleMute", "SetVolume", "SetAudioStreamIndex", "SetSubtitleStreamIndex", "DisplayContent", "GoToSearch", "DisplayMessage", "SetRepeatMode", "ChannelUp", "ChannelDown", "PlayMediaSource", "PlayTrailers"],
+            SupportsPersistentIdentifier: "cordova" === self.appMode || "android" === self.appMode,
+            SupportsMediaControl: true
+        };
+        caps.IconUrl = appHost.deviceIconUrl();
+        caps.SupportsSync = appHost.supports("sync");
+        caps.SupportsContentUploading = appHost.supports("cameraupload");
+        appHost.getPushTokenInfo();
+        return caps = Object.assign(caps, appHost.getPushTokenInfo());
+    }
+};
+var AppInfo = {};
+!function () {
```

Comment:
For what i searched in internet, is used to invoke a function without parenthesis around the function definition. Also it returns true. This can be changed to include the parenthesis and run the function.

Reply:
or remove the function and just run the code inside of it, but if that functions defines a scope, don't know how the app would hold.